raiden_contracts/deploy/contract_deployer.py

```diff
@@ -211,6 +212,56 @@ def _deploy_and_remember(
             address=deployed_contracts['contracts'][contract_name]['address'],
         )
 
+    def register_token_network(
+            self,
+            token_registry_abi: Dict,
+            token_registry_address: str,
+            token_address: str,
+            channel_participant_deposit_limit: Optional[int],
+            token_network_deposit_limit: Optional[int],
+    ):
+        """Register token with a TokenNetworkRegistry contract."""
+        with_limits = contracts_version_expects_deposit_limits(self.contracts_version)
+        if with_limits:
+            assert channel_participant_deposit_limit is not None, \
+                'contracts_version 0.9.0 and afterwards expect channel_participant_deposit_limit'
+            assert token_network_deposit_limit is not None, \
+                'contracts_version 0.9.0 and afterwards expect token_network_deposit_limit'
+        else:
+            assert channel_participant_deposit_limit is None, \
+                'contracts_version below 0.9.0 does not expect channel_participant_deposit_limit'
+            assert token_network_deposit_limit is None, \
+                'contracts_version below 0.9.0 does not expect token_network_deposit_limit'
+        token_network_registry = self.web3.eth.contract(
+            abi=token_registry_abi,
+            address=token_registry_address,
+        )
+
+        version_from_onchain = token_network_registry.functions.contract_version().call()
+        assert version_from_onchain == self.contract_manager.version_string(), \
+            f'got {version_from_onchain}, expected {self.contract_manager.version_string()}'
+
+        command = token_network_registry.functions.createERC20TokenNetwork(
+            token_address,
+            channel_participant_deposit_limit,
+            token_network_deposit_limit,
+        ) if with_limits else token_network_registry.functions.createERC20TokenNetwork(
+            token_address,
+        )
+        self.transact(command)
+
+        token_network_address = token_network_registry.functions.token_to_token_networks(
+            token_address,
+        ).call()
+        token_network_address = to_checksum_address(token_network_address)
+
+        print(
```

Comment:
Should this be a `log` call?

Reply:
Yes, I think so.  I'll change this in this PR.